grab_all_files.py

```diff
@@ -20,7 +20,7 @@ def send_request():
               config or the setup script
 
     '''
-    os.chdir('/home/clay/Downloads/stlCity')
+    os.chdir('/home/clay/Downloads/stl_city')
```

Comment:
updated this for naming consistency, will remove this when all hardcoded filepaths are removed from the module and os env variables used instead.